eunkyung/controllers/postController.js

```diff
@@ -0,0 +1,35 @@
+const postService = require('../services/postService')
+
+
+const posting = async (req, res) => {
+    const { title, content, postImageUrl, userId } = req.body
+    await postService.posting(title, content, postImageUrl, userId)
+    return res.status(201).json({ message: 'POSTING_SUCCESS' })
+}
+
+const userPost = async (req, res) => {
+    const { userId } = req.params;
+    const [userPostResult] = await postService.userPost(userId)
+    return res.status(201).json({ data: userPostResult })
+}
+
+const postList = async (req, res) => {
```

Comment:
함수명을 수정해주세요!
`전체 게시글을 조회한다.`
```suggestion
const getPosts = async (req, res) => {
```